web/src/lib/components/NavTab.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+    const dispatch = createEventDispatcher();
+    export let items;
+    export let activeItem;
+</script>
+
+<div class="mb-5 mt-5">
+    <ul class="mt-0 flex justify-center p-0 list-none">
+        {#each items as item}
+            <li class="mt-0 mr-16 text-lg cursor-pointer" on:click={() => dispatch('tabChange', item)}>
+                <div class:active={item === activeItem}>{item}</div>
```

Comment:
@ibz is there a way to do conditional css classes with tailwind? I couldnt figure this one out so I left the css class below to keep it working - the rest was converted to tailwind

Reply:
Conditional classes can be done using Svelte.

For example, this will add pb-2 (padding bottom 2) to the active item: `class:pb-2={item === activeItem}`